src/lib.rs

```diff
@@ -58,13 +58,12 @@ pub fn clean(path: &str) -> String {
 /// If the result of this process is an empty string, return the string `"."`, representing the current directory.
 fn clean_internal(path: &[u8]) -> Vec<u8> {
     static DOT: u8 = b'.';
-    static SEP: u8 = b'/';
 
     if path.is_empty() {
         return vec![DOT];
     }
 
-    let rooted = path[0] == SEP;
+    let rooted = is_separator(path[0] as char);
```

Comment:
This works for the absolute path to the current drive, but maybe we should support specifying the drive as well? `C:\`